src/olympia/devhub/helpers.py

```diff
@@ -115,7 +115,17 @@ def docs_breadcrumbs(context, items=None):
 @jinja2.contextfunction
 def add_file_modal(context, title, action, action_label, modal_type='file'):
     addon = context['addon']
-    upload_url = reverse('devhub.upload_for_addon', args=[addon.slug])
+    version = context.get('version',
+                          addon.find_latest_version_including_rejected())
```

Comment:
Side note: I had jinja helpers that pull stuff magically from the context. I know it was already pulling `addon`, but that sucks :/